### PR TITLE
 Trim whitespaces at the end of line in es_ES Messages.properties file (JENKINS-58842) workaround

### PR DESCRIPTION
**Pull request by diegombeltran**
Trim whitespaces at the end of line.

See [JENKINS-58842](https://issues.jenkins-ci.org/browse/JENKINS-58842).

### Proposed changelog entries

* Entry 1: Trim whitespaces at the end of line in es_ES Messages.properties file

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
The PR title does not represent what the PR really does. It makes sense to trim spaces, but it has nothing to do with IAM roles

**Comment by MRamonLeon** [comment]
Hello @diegombeltran . First of all, thank you for your contribution, appreciated. Welcome to the Jenkins community.

I see a lot of changes in the diff that effectively doesn't change anything. Could you please remove these changes from the PR to leave it cleaner? And change the title as Oleg suggested?
Example:
The text is the same as before:
_ResultTrend.StillFailing=Todavía falla_ -> _ResultTrend.StillFailing=Todavía falla_ 

Have you checked that the messages in other languages (English for example) don't have these blanks at the end? Maybe the message is concatenated to another one, so the blank at the end is needed.

Gracias!

**Review by oleg-nenashev — APPROVED** [review]
https://github.com/jenkinsci/aws-credentials-plugin/pull/69 for a proper fix of the issue. I do not mind merging this pull request as well since it really cleans up the localization. But it is definitely not a root cause of JENKINS-58842

File: core/src/main/resources/hudson/model/Messages_es.properties [hunk 1]
```diff
@@ -120,11 +120,11 @@ Job.NOfMFailed={0} de las {1} \u00faltimas ejecuciones fallaron.
 Job.NoRecentBuildFailed=No hay ejecuciones recientes con fallos.
 Job.Pronoun=Proyecto
 Job.minutes=Min
-Job.NoRenameWhileBuilding=No es posible renombrar un proyecto mientras se est� ejecutando.
+Job.NoRenameWhileBuilding=No es posible renombrar un proyecto mientras se está ejecutando.
```
Comment (jglick) [inline core/src/main/resources/hudson/model/Messages_es.properties]: GitHub refuses to show a diff here, but it seems you changed the encoding to UTF-8. It must be ISO-8859-1. Acc. to `jshell`,

```java
var p = new Properties()
p.load(new java.io.FileInputStream("/…/jenkinsci/jenkins/core/src/main/resources/hudson/model/Messages_es.properties"))
p.getProperty("Job.NoRenameWhileBuilding")
```

returns

```
No es posible renombrar un proyecto mientras se está ejecutando.
```

while the equivalent passing in the file as patched here produces mojibake

```
No es posible renombrar un proyecto mientras se estÃ¡ ejecutando.
```

Safest to throw away this modification and start over with an editor or tool that can safely handle ISO-8859-1 text.

**Review by jglick — CHANGES_REQUESTED** [review]

**Comment by jglick** [comment]
Note that Java supports loading `*.properties` in an encoding of your choice, but this is worse than nothing IMO because it does so based on a parameter passed by the Java code loading the file, not by inspecting the file for some magic header or BOM or something that could also be interpreted by editors and other tooling; so editors must assume that the file uses the historical default of ISO-8859-1. For example, the NetBeans editor always loads `*.properties` in this encoding, regardless of project-wide defaults.

Perhaps we should deprecate use of `java.util.Properties` for `Messages` classes (and other usages, like Stapler localization of `*.jelly`) altogether, and switch to some file format more familiar to the youth of today which is defined to be UTF-8.

**Comment by MRamonLeon** [comment]
> I see a lot of changes in the diff that effectively doesn't change anything. Could you please remove these changes from the PR to leave it cleaner?

That should be the problem, GitHub is writing the same text, no matter which encoding the file has. Please double-check the encoding of the file.

**Comment by diegombeltran** [comment]
Hello,

First of all thank you for spending time on this. I'm currently on my holidays far from computers, I will check this on Saturday or so.

See you!

**Comment by jglick** [comment]
> deprecate use of `java.util.Properties` for `Messages` classes (and other usages

See https://github.com/kohsuke/localizer/pull/20 for example. CC @ikedam

**Comment by varyvol** [comment]
@diegombeltran did you eventually have the opportunity to spend some time on this? I'm going to propose it for close but please feel free to reach in case you think otherwise.

**Comment by rsandell** [comment]
No response from author, so closing.

**Pull request closed without merging**